bench_runner/templates/workflow_bootstrap.py

```diff
@@ -0,0 +1,79 @@
+# This script may only use the standard library, since it bootstraps setting up
+# the virtual environment to run the full bench_runner.
+
+
+# NOTE: This file should import in Python 3.9 or later so it can at least print
```

Comment:
Later we require Python 3.11 though

Reply:
Oh, actually, this is correct.  `bench_runner` requires 3.11 and above and is only tested on that.  However, the oldest LTS Linux we use installs Python 3.9 as default.  It's nice to be able to run this script and get a nice error message that a Python 3.11 needs to be installed.